flow/workflows/sync_flow.go

```diff
@@ -85,6 +85,11 @@ func SyncFlowWorkflow(
 		})
 	}
 
+	syncFlowCtx := workflow.WithActivityOptions(syncSessionCtx, workflow.ActivityOptions{
```

Comment:
what's the rationale for moving this up?

Reply:
less allocations, unrelated